### DISCUSSION TITLE
transparency slider is obscured

### DISCUSSION THREAD

**Discussion by Wonderer8** [osmandapp/OsmAnd#18407] [Ideas]
I have been an enthusiastic user of OSMAND for several years. 
The transparency slider for the underlay/overlay map is obscured by the zoom level indicator for a very long time after you change it. 
Is it possible not to position both in the same place? 
Regards

**Comment by DmitryAlexei** [comment]
@Wonderer8 to better understand your issue, could you please add a screenshot
